eng/mgmt/automation/sdk_init.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/sh
 
-sudo apt-get install -y --upgrade python3-pip python3-setuptools
-pip3 install --upgrade wheel
-pip3 install --upgrade PyYAML requests
+apt-get install python3.8
+apt-get install -y --upgrade python3-pip python3-setuptools
```

Comment:
Is this works same on sdkautomation?

Reply:
sdk_init.sh only used in sdk generation pipeline docker. Docker run script in root user, so we don't need sudo cmd. Script has tested in pipeline docker and local docker.